charts/kargo/templates/controller/cluster-roles.yaml

```diff
@@ -93,26 +93,4 @@ rules:
   - patch
   - watch
 {{- end }}
-{{- if .Values.controller.rollouts.integrationEnabled }}
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: kargo-controller-rollouts
-  labels:
-    {{- include "kargo.labels" . | nindent 4 }}
-    {{- include "kargo.controller.labels" . | nindent 4 }}
-rules:
-- apiGroups:
-  - argoproj.io
-  resources:
-  - analysisruns
```

Comment:
I think these permissions are still required by the verification process?

Reply:
Odd... this all _worked_. Let me look into that.

Reply:
Oh... I removed those last minute _after_ I saw it working. Fixing.